webpack.config.js

```diff
@@ -82,7 +81,11 @@ const webpackConfig = {
 			{
 				test: /\.jsx?$/,
 				exclude: /node_modules[\/\\](?!notifications-panel)/,
-				loader: [ 'happypack/loader' ]
+				loader: _.compact( [
+					'thread-loader',
```

Comment:
If you want to compare apples-to-apples, `happypack` uses 3 threads by default (which, in my tests, seemed to be the sweet spot), and `thread-loader` as many as CPU logical cores (which means 8 in a 15" MBP).

In both cases it's configurable, so I suggest putting the same number of threads in both cases to have an idea of which one is better :)

Reply:
I grew less inspired with this PR when i didn't see a big difference.  I'll close it